docs/src/components/Heading/Heading.scss

```diff
@@ -38,12 +38,12 @@ import BpkHorizontalNav, {
   }
 
   &--h2 {
-    margin-bottom: $bpk-spacing-xxl;
+    margin-bottom: 1.125rem;
 
     @include bpk-text-xxl;
 
     @include bpk-breakpoint-mobile {
-      margin-bottom: $bpk-spacing-base;
+      margin-bottom: 1.5rem;
```

Comment:
Shouldn't we use something derived from Backpack tokens here?

Reply:
Tru dat. Fixed